src/features/navigate/Navigate.tsx

```diff
@@ -0,0 +1,103 @@
+import './Navigate.css';
+import img from '../../assets/logo.png';
+import {
+  AppBar,
+  Box,
+  Button,
+  Tab,
+  Tabs,
+  Toolbar,
+  Typography,
+} from '@mui/material';
+import { useEffect, useState } from 'react';
+import { Outlet, useNavigate } from 'react-router-dom';
+import { useAccount, useDisconnect } from 'wagmi';
+import { useWeb3Modal } from '@web3modal/react';
```

Comment:
how `wagmi` is supposed to work with `web3modal`?

Reply:
the link between the two library is made in the main.tsx
``